### DISCUSSION TITLE
Include number of unsolicited connections browsers make

### DISCUSSION THREAD

**Discussion by avidseeker** [privacytests/privacytests.org#215] [Ideas]
These links can be helpful for this task:
* [Brave](https://spyware.neocities.org/articles/brave)
* [Chrome](https://spyware.neocities.org/articles/chrome)
* [Firefox](https://spyware.neocities.org/articles/firefox)
* [Librewolf](https://spyware.neocities.org/articles/librewolf)
* [Ungoogled Chromium](https://spyware.neocities.org/articles/ungoogled_chromium)

**Comment by brinerustle** [comment]
Those links are all very old, though the point is valid. Question is, how do you quantify it?
